chatcommands.py

```diff
@@ -1731,7 +1731,7 @@ def allspam(msg, url):
         # Fetch posts
         request_url = "https://api.stackexchange.com/2.2/users/{}/posts".format(u_id)
         params = {
-            'filter': '!)Q4RrMH0DC96Y4g9yVzuwUrW',
+            'filter': '!*0S0--txpIwJHBnEqJR9px_-b(4TI_dAeSMiPUcCA',
```

Comment:
This filter does not appear to contain `post/body_markdown`, which is what you are trying to get here, at least as far as I understand.

Reply:
Oops, I only looked into `answer` and `question` and even wondered for a while why it included neither `answer/body` nor `question/body`.

Reply:
Fixed in abd4b5011ee82b206586411b11b616ca7fdb832f